examples/research_projects/layer_skip/scripts/benchmark_layer_skip.py

```diff
@@ -0,0 +1,63 @@
+import config
+import torch
+from torch.utils import benchmark
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+
+def generate_tokens(model, inputs):
+    outputs = model.generate(
+        **inputs,
+        do_sample=False,
+        max_new_tokens=64,
+    )
+    return outputs
+
+
+def generate_assistant_tokens(model, inputs, assistant_early_exit):
```

Comment:
I suggest to name this function `generate_tokens_with_asssistance` because it does generate "final" tokens, and generates assistant tokens under the hood.
```suggestion
def generate_tokens_with_assistance(model, inputs, assistant_early_exit):
```